docs/reference/kusk-cli.md

```diff
@@ -0,0 +1,206 @@
+# Kusk CLI
+
+Kusk is a CLI tool designed to help you manage common tasks required when
+running Kusk Gateway.
+
+Currently we support the following commands:
+- `install` - installs Kusk Gateway and all its components with a single command. (Requires a helm installation)
+- `api generate` - for creating Kusk Gateway API resources from your OpenAPI specification document.
+
+---
+
+# Table of contents
+- [Installation](#installation)
+- [Updating](#updating)
+- [Commands](#commands)
+  - [install](#install)
+  - [api generate](#api-generate)
+- [Uninstallation](#uninstallation)
+- [Contributing](#contributing)
+- [License](#license)
+
+# Installation
+
+[(Back to top)](#table-of-contents)
```

Comment:
I'd remove these back-to-tops as it's not in the rest of our docs and they still have the page nav